monitor.py

```diff
@@ -476,13 +476,13 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         else:
             nightly = job_platform+"-latest-"+ nightly_log_match.group(2)
     else:
-        nightly_initial_log_re = re.compile('(Resolved release {}-initial to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
+        nightly_initial_log_re = re.compile('(Using explicitly provided pull-spec for release {}-initial \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
```

Comment:
This changes will work only for release controller triggered upgrade jobs. You should probably add this in a else condition to support both